api/src/wfl/executor.clj

```diff
@@ -307,16 +314,17 @@
   "Create new submission from new `source` snapshot if available,
   writing its workflows to `details` table.
   Update statuses for active or failed workflows in `details` table.
-  Return `executor`."
-  [source executor]
+  Return updated `workload`."
+  [{:keys [executor source] :as workload}]
   (when-let [object (stage/peek-queue source)]
-    (let [entity     (from-source executor object)
-          submission (create-submission! executor entity)]
+    (let [entity      (from-source executor object)
+          userComment (create-user-comment "New submission" workload)
```

Comment:
Good thinking to distinguish automatically triggered submissions from manually triggered retries.

Most recently, I triggered a COVID retry that Christine didn't know about, and that caused some confusion.  So this will help!